pkg/controllers/v1alpha1/jindo/jindoruntime_controller.go

```diff
@@ -101,9 +105,27 @@ func (r *RuntimeReconciler) Reconcile(context context.Context, req ctrl.Request)
 }
 
 //SetupWithManager setups the manager with RuntimeReconciler
-func (r *RuntimeReconciler) SetupWithManager(mgr ctrl.Manager, options controller.Options) error {
-	return ctrl.NewControllerManagedBy(mgr).
-		WithOptions(options).
-		For(&datav1alpha1.JindoRuntime{}).
-		Complete(r)
+func (r *RuntimeReconciler) SetupWithManager(mgr ctrl.Manager, options controller.Options, eventDriven bool) (err error) {
```

Comment:
Will eventDriver be standard for all Fluid Runtime?

Reply:
Yes, it will be standard later. But we will use flag to make it be able for downgrade.